certified-connectors/Converter/apiProperties.json

```diff
@@ -0,0 +1,22 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "Api-Key (free on \"https://power2apps.de/apikey\")",
+          "description": "\"Api-Key (free on \"https://power2apps.de/apikey\")\" f\u00fcr diese API",
+          "tooltip": "Stellen Sie \"Api-Key (free on \"https://power2apps.de/apikey\")\" bereit.",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#FFFFFF",
+    "capabilities": [],
+    "publisher": "Siek, Stephan"
```

Comment:
@stsiek Please add the stackOwner property.

https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-3a-publisher-and-stackowner-properties